nilearn/datasets/utils.py

```diff
@@ -576,13 +576,8 @@ def _fetch_file(url, data_dir, resume=True, overwrite=False,
             sys.stderr.write(' ...done. ({0:.0f} seconds, {1:.0f} min)\n'
                              .format(dt, dt // 60))
     except (_urllib.error.HTTPError, _urllib.error.URLError) as e:
```

Comment:
minor comment: you don't need the `as e` since you won't do anything with `e`; pyflakes might complain about an unused variable